src/sass/layout/_hero.scss

```diff
@@ -1,13 +1,19 @@
 .hero__section {
+  position: relative;
   display: flex;
```

Comment:
please, use a mixin "flex".

Reply:
changed to mixins (flex and font) where it was possible